app/components/PhysicsPlane.js

```diff
@@ -0,0 +1,42 @@
+import {
+  MeshPhongMaterial
+} from 'three';
+
+import {
+  PlaneModule
+} from 'physics-module-ammonext';
+
+// Meshes
+import {
+  Plane
+} from '@whs+meshes';
+
+import {MeshComponent} from '@whs/core/MeshComponent';
+
+export class PhysicsPlane extends Plane {
+  /**
+   * @name PhysicsPlane
+   * @param params
+   */
+  constructor(params = {}) {
+
+    params.modules = [
+      new PlaneModule({
```

Comment:
That will overwrite all `modules` array. **I'd recommend** using `this.applyModule()` after calling `super(...)`.

```javascript
this.applyModule(
  new PlaneModule({
    mass: 0
  })
);
```